components/concept/engine/src/main/java/mozilla/components/concept/engine/ClearDatable.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine
+
+/**
+ * Contract to indicate how objects able to clear browser data should behave.
```

Comment:
nit: Contract to indicate how objects with ability to clear data should behave?

Reply:
👍